document-sandbox-samples/image-and-page/src/code.js

```diff
@@ -32,20 +32,17 @@ async function start() {
             const rectangle = editor.createRectangle();
             rectangle.width = 200;
             rectangle.height = 150;
-            rectangle.translateX = 100;
-            rectangle.translateY = 20;
```

Comment:
It looks like express-grids-addon/grids-design-end/src/documentSandbox/shapeUtils.js also uses `translateX` and needs to be updated, fyi.

Reply:
Also btw, are you doing any manual testing of these sample add-ons to make sure they work with all the PR's changes? Since they're all in .js and there's no build process or type checking, we have way less safety net here than for the regular code changes we're used to working on...

Reply:
@vamshich13 , @peterflynn I'm on the renaming for the grids add-on—which I suspect also needs some revising for the new way colors are created with object literals.

Reply:
ok thanks @undavide 
so Is it fine to merge this PR before without changes in grid add-ons ?
if later you will create a separate PR for grid-addon changes.

Reply:
Yes @vamshich13, I will do a PR to the doc website (tutorial) and a PR here (sample).